### DISCUSSION TITLE
Map without background

### DISCUSSION THREAD

**Discussion by ssundell** [time4tea/gopro-dashboard-overlay#69] [Ideas]
Hi!

Would it be possible to have the map component without background? In essence the journey map with the route marked and the dot showing the position in the route, but without the base map.

I'm still trying out the different styles, and there are plenty, but it's just something that I'm somewhat used to 8) 

The overlays look great, and I'm eagerly waiting for the gpx only creation of the graphics 8)

**Comment by time4tea** [comment]
Hiya - This should be do-able. I'm not giving promises about when features will happen - but this one sounds like a simple version of it would be not too tricky.

**Reply by ssundell** [reply]
Thanks, that sounds excellent - I'm in no hurry, I'm just glad to use the software 8)

**Comment by time4tea** [comment]
This is in version 0.69.0 as a component "circuit_map" - it is very basic. Please let me know if its any use.
There are instructions here: https://github.com/time4tea/gopro-dashboard-overlay/tree/main/docs/xml/examples/06-circuit-map
